apps/src/applab/firebaseUtils.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+/* global Applab */
+
+let Firebase = require("firebase");
+
+let databaseCache = {};
+
+export default {
+  getDatabase(channelId) {
```

Comment:
I think the better approach here (assume you're going to have more utils in the future) would be to just:
```
export function getDataBase(channelId) {
...
}
```

And then in the file where you use it:
```
import { getDatabase } from './firebaseUtils';
...
getDatabase(channelId);